src/components/filter/IsSuccesfulLanding.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+export default function isSuccesfulLanding(props) {
+  const toggleSuccessFull = (val) => {
+    props.filter.successfullLanding === val ?
+    props.setFilter({ ...props.filter, successfullLanding: "" })
+    : props.setFilter({ ...props.filter, successfullLanding: val });
+  }
```

Comment:
```suggestion
  const toggleSuccessFull = useCallback(() => {
    props.filter.successfullLanding === val ?
    props.setFilter({ ...props.filter, successfullLanding: "" })
    : props.setFilter({ ...props.filter, successfullLanding: val });
  }, [val])
```

bcz whenever the function is called it will create a new `toggleSuccessFull` function.

so use either `useCallback` or `useMemo`